qemu/tests/cfg/scsi_mq_check.cfg

```diff
@@ -0,0 +1,14 @@
+- scsi_mq_check:
+    # As WinXP, Win2003, Win2008, Win7 not support multi queue. So don't test them.
+    no WinXP Win2003 Win2008 Win7
+    no RHEL.4 RHEL.5 RHEL.6
+    only virtio_scsi
+    type = scsi_mq_check
+    driver_name = vioscsi
+    # for linux:
+    pattern = "PCI-MSI-edge\s+virtio0-request"
```

Comment:
For ppc64le, the pattern is not available, no keyword "PCI-MSI-edge" on power platform.
eg:
[root@dhcp71-104 ~]# cat /proc/interrupts 
           CPU0       CPU1       CPU2       CPU3       
 16:       2424       3516       4753       3581      XICS   2 Edge      IPI
 19:          0          0          0          0      XICS 4096 Edge      RAS_EPOW
 21:         30          0          0          0      XICS 4103 Edge      hvc_console
 22:          0          0          0          0      XICS 4097 Edge      RAS_HOTPLUG
 23:          0          0          0          0      XICS 4104 Edge      virtio0-config
 24:          0          0          0          0      XICS 4105 Edge      virtio0-control
 25:          0          0          0          0      XICS 4106 Edge      virtio0-event
 26:       6138          0          0          0      XICS 4107 Edge      virtio0-request
 27:          0          0          0          0      XICS 4108 Edge      virtio1-config
 28:        479          0          0          0      XICS 4109 Edge      virtio1-input.0
 29:          1          0          0          0      XICS 4110 Edge      virtio1-output.0
LOC:       1884       1473       3416       1532   Local timer interrupts for timer event device
LOC:          1          1          0          0   Local timer interrupts for others
SPU:          0          0          0          0   Spurious interrupts
PMI:          0          0          0          0   Performance monitoring interrupts
MCE:          0          0          0          0   Machine check exceptions
NMI:          0          0          0          0   System Reset interrupts
WDG:          0          0          0          0   Watchdog soft-NMI interrupts
DBL:          0          0          0          0   Doorbell interrupts

Reply:
Hi yongxue, how about set pattern = "virtio\d+-request" for ppc64le?
I tried on x86 platform, it can be passed.

Reply:
@YongxueHong Hi Yongxue, do you agree peixiu's suggestion?

Reply:
@miyachen @peixiu 
Sorry for replaying later, it's ok for ppc64le.
thanks.